Team.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <iostream>
+#include <string>
+#include <iomanip>
+
+class Team
+{
+public:
+	std::string m_name;
```

Comment:
Again, m_ is prefix for private variables and you could've just made Team a struct since all it members are public.